### ISSUE TITLE
Official 3DS Port?

### ISSUE THREAD

**Issue by TheNamesGregoryAndounuts** [ppy/osu#10529]
**Describe the new feature:** 
Okay, so I'm a fan of the the original Ouedan and Elite Beat Agents games that Osu is based on, so I'm used to that style of play: Tapping the beat markers with a stylus to play maps. But currently I find myself struggling to adapt to playing with the mouse. The act of clicking beat markers and holding on the click to keep up phrase markers while simultaneously having to move the mouse around is too cumbersome. The gameplay doesn't feel as fluid as playing with the stylus. (See the following videos to get a feel for what I mean https://www.youtube.com/watch?v=C7y4EmzsKR0 and https://www.youtube.com/watch?v=SyAxIugbvRI& This guy is having a horrible time keeping the beat because the Osu/EBA style of gameplay wasn't designed for anything but the DS' touchscreen.) So I would like to see official support for a 3DS port so I can enjoy Osu how it was meant to be played. 
**Proposal designs of the feature:** 
Basically Osu 3DS will play just like standard PC Osu. Only now, gameplay is split across the dual screens, with the score counter and storyboard on top, and the beat map on the bottom, so it's more closely resembles EBA and OTO. Maybe we could set it up so each screen has it's own storyboard for those mappers who want to go whole hog and create true iNiS style maps.

I hope your can give my suggestion your full consideration.

**Comment by peppy** [comment]
https://github.com/echojc/osu-ds This is the closest you'll get.

lazer is not made to run on the 3DS.
